config/environments/production.rb

```diff
@@ -87,6 +87,13 @@
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
 
+  config.imap_settings = {
```

Comment:
wherebot.rb doesn't seem to use this config, it just looks at the environment variables directly. Can we remove this? (and the similar lines in development.rb)